notebooks/Notebook2.md

```diff
@@ -17,18 +12,34 @@ exciting milestones, preliminary results, etc. Feel free to include images
 (e.g., a sketch of the design or a screenshot of a running program), links to
 code, and any other resources that you think will help clearly convey your
 design process.
+-- REMOVE 
+Some reps to look at https://github.com/hmc-cs111-spring2023/hw-7-piconot-rydermitchell-hw7/blob/main/build.md
+https://github.com/hmc-cs111-spring2023/hw-7-sample-solution/tree/main/src/main/scala/piconot/external
+
+--
+
+This week I did not work as much as I would have liked. I mainly just took our sample solution from hw7 and converted it into my artifact repo. I want to use its external solution as structure for my own.  The main progress I made was on a language for parseing my language. Check it out in artifact [design.md](https://github.com/hmc-cs111-spring2023/artifact-AdamrBeckwith/blob/main/design.md) would love feedback on that setup. (I am considering the writing there as part of this notebook) 
```

Comment:
Overall, I like the design, from what I understand of the syntax! There are lots of elements that are composable, and I really how you have described them (e.g., that 5d6 is 5 instances of 1d6 or that a check is a roll + a condition, which has type boolean, or `=> boolean`).